components/binding/cmd/manager/main.go

```diff
@@ -93,3 +105,28 @@ func fatalOnError(err error, msg string) {
 		log.Fatalf("%s: %s", msg, err.Error())
 	}
 }
+
+// syncTargetKinds assures that existing TargetKinds will be loaded before reconcilers are started
```

Comment:
please move it to a separate file (inside `internal`), let's avoid a situation when the main file grows all the time

Reply:
didn't find a better place than `/internal/syncer/syncer.go` as a separate struct, now execution looks like:
 ```
err = syncer.NewExecutor(cli, logger).TargetKinds(targetKindWorker)
```